### PR TITLE
Update use case in developer guide

### PR DESCRIPTION
**Pull request by hweinian**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -473,6 +472,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
    Use case ends
 
+**Extensions:**
+
+* 2a. Medibase is unable to list patient details
+
+    * 2a1. MediBase3 does not show any patients.
+
+      Use case ends.
```
Comment (yong-zhuo) [inline docs/DeveloperGuide.md:481]: Appropriate extension 👍🏻

**Review by yong-zhuo — APPROVED** [review]
Overall LGTM, do ensure that the 'use case end' is placed on the line below the last step and not with the last step

**Review by PresleyChew — APPROVED** [review]
Overall LGTM, but wait for 1 more reviewer as use case has a lot of details we don't want to miss out on.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -430,17 +430,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS:**
 1. User requests to find a patient by NRIC
-2. MediBase3 request for search requirements
-3. User provides the details required to search for the patient
-4. MediBase3 shows the selected patient information
+2. User provides the details required to search for the patient
+3. MediBase3 shows the selected patient information
```
Comment (TechRaj) [inline docs/DeveloperGuide.md:434]: good catch

**Review by TechRaj — APPROVED** [review]
LGTM

**Pull request merged**